.chloggen/datadogreceiver-span-obfuscate.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
```

Comment:
What is this file and where does it go?

Reply:
Oh, this is the change log used when publishing new versions in the official repo. We don't need it here; it just came from the rebase of my PR initially created there.